agreementbot/consumer_protocol_handler.go

```diff
@@ -535,6 +546,13 @@ func (b *BaseConsumerProtocolHandler) HandlePolicyChangeForAgreement(ag persiste
 		}
 	}
 
+	if same, msg := consumerPol.IsSamePolicy(oldPolicy); same {
+		glog.V(3).Infof("business policy(producerPol) %v content remains same with old policy", ag.PolicyName)
+		return true, true, true
+	} else {
+		glog.V(3).Infof("business policy %v content changed: %v", ag.PolicyName, msg)
```

Comment:
Can you add the agreementid here too similar to above but that this agreement does need the update?

Reply:
updated